adapters/InMobi/InMobiAdapter/GADMAdapterInMobi.m

```diff
@@ -128,10 +128,9 @@ - (void)prepareRequestParameters {
   GADMAdapterInMobiMutableDictionarySetObjectForKey(dict, @"tp", @"c_admob");
   GADMAdapterInMobiMutableDictionarySetObjectForKey(dict, @"tp-ver", [GADRequest sdkVersion]);
 
-  if ([[self.connector childDirectedTreatment] integerValue] == 1) {
-    GADMAdapterInMobiMutableDictionarySetObjectForKey(dict, @"coppa", @"1");
-  } else {
-    GADMAdapterInMobiMutableDictionarySetObjectForKey(dict, @"coppa", @"0");
+  if ([self.connector childDirectedTreatment]) {
+    NSString *coppaString = [[self.connector childDirectedTreatment] integerValue]? @"1" : @"0";
+    GADMAdapterInMobiMutableDictionarySetObjectForKey(extrasDictionary, @"coppa", coppaString);
```

Comment:
compiling error: Use of undeclared identifier 'extrasDictionary'

Reply:
Changes done.